themes/SuiteP/css/yui.css

```diff
@@ -958,8 +958,8 @@ span.yui-carousel-first-button-disabled {
 
 .yui-panel {
     position: relative;
-    left: 0;
```

Comment:
Again, these are very specific....  Would this work with or without the sidebar?  or on a larger screen?